src/app/pages/receive/components/receive-items.tsx

```diff
@@ -7,8 +7,12 @@ interface ReceiveItemListProps {
 export function ReceiveItemList({ children, title }: ReceiveItemListProps) {
   return (
     <>
-      {title && <styled.span textStyle="">{title}</styled.span>}
-      <Divider mt="space.02" />
+      {title && (
```

Comment:
@mica000 - hopefully this looks OK now. The font style wasn't being set either

![Screenshot 2023-11-22 at 09 23 03](https://github.com/leather-wallet/extension/assets/2938440/1cd2a4be-d67d-4a5b-ba5b-2e93f8b0d8e1)